Content.Server/AlertLevel/Systems/AlertLevelChangeOnTriggerSystem.cs

```diff
@@ -0,0 +1,39 @@
+using Content.Server.AlertLevel;
+using Content.Server.Explosion.EntitySystems;
+using Content.Server.Station.Systems;
+
+
+namespace Content.Server.AlertLevel.Systems;
+
+public sealed class AlertLevelChangeOnTriggerSystem : EntitySystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<AlertLevelChangeOnTriggerComponent, TriggerEvent>(OnTrigger);
+    }
+
+    [Dependency] private readonly AlertLevelSystem _alertLevelSystem = default!;
+    [Dependency] private readonly StationSystem _station = default!;
+
+    private void OnTrigger(EntityUid uid, AlertLevelChangeOnTriggerComponent ent, ref TriggerEvent args)
```

Comment:
```suggestion
    private void OnTrigger(Entity<AlertLevelChangeOnTriggerComponent> ent, ref TriggerEvent args)
```
use `ent.Comp` and `ent.Owner` below